db/migrate/20220610054628_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> As the project requires, could you make sure that user on the post model is referenced as an author, something like this: `t.references :author, foreign_key: { to_table: 'users' }`

Reply:
@rukundoeric Thanks a lot for suggestion I will fix it.